src/components/expired-subscription-modal/index.jsx

```diff
@@ -1,11 +1,11 @@
 import {
-  useToggle, AlertModal, ActionRow, StatefulButton,
+  ActionRow, AlertModal, StatefulButton, useToggle,
 } from '@openedx/paragon';
 import DOMPurify from 'dompurify';
 import { useState } from 'react';
 import { postUnlinkUserFromEnterprise, useEnterpriseCustomer, useSubscriptions } from '../app/data';
 
-const ExpiredSubscriptionModal = () => {
+const BlockingSubscriptionExpirationModal = () => {
```

Comment:
nit: should the folder name containing this component be updated to reflect the changed component name?